core/metrics-core-api/pom.xml

```diff
@@ -52,6 +52,11 @@
     </dependency>
 
     <dependency>
+      <groupId>io.reactivex</groupId>
+      <artifactId>rxjava-math</artifactId>
```

Comment:
We do because Aggregate is in the API and the the predefined aggregates use MathObservable.